zhaquirks/tuya/mcu/__init__.py

```diff
@@ -740,6 +750,24 @@ class TuyaSoilMoisture(SoilMoisture, TuyaLocalCluster):
     """Tuya local SoilMoisture cluster with a device RH_MULTIPLIER factor if required."""
 
 
+class TuyaValveWaterConsumed(Metering, TuyaLocalCluster):
+    """Tuya Valve Water consumed cluster."""
+
+    VOLUME_LITERS = 0x0007
+    WATER_METERING = 0x02
+
+    """Setting unit of measurement."""
+    _CONSTANT_ATTRIBUTES = {
+        0x0300: VOLUME_LITERS,
+        0x0306: WATER_METERING,
+    }
```

Comment:
We should be able to use zigpy `AttributeDefs` for this. Example:
```python
    _CONSTANT_ATTRIBUTES = {
        Metering.AttributeDefs.unit_of_measure.id: attr_value,
        Metering.AttributeDefs.metering_device_type.id: attr_value,
    }
```
It looks like "units of measure" and "metering device types" do not have their own classes in zigpy at the moment, so the constants for `VOLUME_LITERS` and `WATER_METERING` still make sense there.